client/state/selectors/is-site-coming-soon.js

```diff
@@ -14,7 +15,7 @@ import isPrivateSite from 'state/selectors/is-private-site';
  * @returns {boolean} True if site is coming_soon
  */
 export default function isSiteComingSoon( state, siteId ) {
-	if ( ! isPrivateSite( state, siteId ) ) {
+	if ( ! isPrivateSite( state, siteId ) && ! isEnabled( 'coming-soon-v2' ) ) {
```

Comment:
This doesn't make sense to me unless you also check `wpcom_public_coming_soon` but I don't think we want to do that (possibly has knock-on effects.)

Can probably just remove it, this is a v1 only function unless something changed.

Reply:
Yeah, I think that private/public test is a little redundant. Coming soon is either on or off, right? I'll remove it and the associated unit tests